modules/reference/pages/command-line-tools.adoc

```diff
@@ -695,6 +695,68 @@ Show help message and exit.
 
 
 
+[[spacewalk-repo-sync]]
+== Synchronize Software Repositories into {productname} Channels (spacewalk-repo-sync)
```

Comment:
Wow, long title. What about:

```suggestion
== Synchronize Software Repositories into Channels
```

Or: 

```suggestion
== Synchronize Repositories  with spacewalk-repo-sync
```

Reply:
I'll take the or version (because it is in the Ref Guide, I'd like to see the name of the tool we are discussing in the title.